Fix some installs and imports

recycletechno opened the pull request:
Several fixes regarding old version of torchtext, some spaCy fixes and proper pip install strings for Google Colab

recycletechno committed:
Fix some installs and imports

nkarpachev reviewed (commented):
Hw01_NMT: why decrease N_EPOCHS in baseline from 10 to 3?

recycletechno commented:
> Hw01_NMT: why decrease N_EPOCHS in baseline from 10 to 3?

I commented here that "3 epochs are for test reasons only. Increase if needed". Yes, I think it is better to revert to 10 here

recycletechno committed:
fixed 10 epochs and set random_state to split

recycletechno commented:
> Hw01_NMT: why decrease N_EPOCHS in baseline from 10 to 3?

Fixed. And also added `random_state` to torch split because it caused some "reproducibility issues" with different vocab sizes.
